checkstyle-style.xml

```diff
@@ -263,6 +266,8 @@
             <property name="tokens" value="VARIABLE_DEF"/>
         </module>
 
+        <module name="SuppressionCommentFilter"/>
```

Comment:
`SuppressionCommentFilter` is moved and now becomes a child of TreeWalker [in new release](http://checkstyle.sourceforge.net/releasenotes.html#Release_8.1)